File: src/test/scala/TestLogs.scala
import org.apache.logging.log4j.LogManager

/**
  * Created by prayagupd
  * on 12/20/16.
  */

object TestLogs {
  val logger =  LogManager.getLogger("fhkajgh")

  def main(args: Array[String]): Unit = {
    logger.info("this world is full of crap people")
  }
}

File: src/main/java/TestLogs.java
import org.apache.logging.log4j.LogManager;
import org.apache.logging.log4j.Logger;

public class TestLogs {

    static Logger logger = LogManager.getLogger(TestLogs.class);

    public static void main(String[] args) {
        logger.info("key:\"move one mate, sooner\"");
    }
}
